### PR TITLE
fix(tests): Fix pytest warnings from upgrade

### PR DESCRIPTION
**Pull request by mattrobenolt**
There are others, but they seem to be from outside our codebase.

### CODE REVIEW

File: src/sentry/utils/pytest/selenium.py [hunk 1]
```diff
@@ -224,9 +224,10 @@ def _environment(request):
     config._environment.append(('Driver', config.option.selenium_driver))
 
 
-@pytest.mark.tryfirst
-def pytest_runtest_makereport(item, call, __multicall__):
-    report = __multicall__.execute()
+@pytest.hookimpl(hookwrapper=True)
```
Comment (mattrobenolt) [inline src/sentry/utils/pytest/selenium.py]: tbh waiting on Travis for this one to confirm it actually worked. I hobbled this together from random PRs that fixed this warning themselves by doing something similar.

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Changes to build requirements</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Review by tkaemming — CHANGES_REQUESTED** [review]
Requesting changes since you mentioned making some additional fixes and this way we don't get emailed about it. 🙂  This change itself looks fine and probably does something.

**Comment by markstory** [comment]
Is this something we're still interested in pursuing or can it be closed because it can no longer be merged?

**Pull request closed without merging**